tools/python/boutiques/bosh.py

```diff
@@ -436,6 +481,12 @@ def bosh_return(val, code=0, hide=False):
         elif func == "test":
             out = test(*params)
             return bosh_return(out)
+        elif func == "search":
+            out = search(*params)
+            return bosh_return(out)
```

Comment:
so in case `search` is called from the Python API, it would return a tabulated string, difficult to parse. It would be better to return a dict in this case. Suggestion:
1. search could return a dict
2. bosh_return could take a formatting function as a parameter, defaulting to `print`, and set to `print(tabulate(x))` in this case.